docs/scripts/copy_md_files.js

```diff
@@ -0,0 +1,29 @@
+/**
+*/
+const fs = require('fs');
+
+const { copyFile } = fs.promises;
+
+const defaultOptions = {
+    files: {},
+    docsDir: './docs'
+};
+
+/**
+    This function is used to copy markdown files from a source
+    outside the `docs/` folder to a destination inside the `docs/` folder.
+    Options:
+    - files:          a mapping of source: destination
+    - docsDir:        the docs folder
+*/
+async function copyMarkdownFiles(options) {
+
+    options = { ...defaultOptions, ...options };
+    const { docsDir, files } = options;
+
+    for (const [source, destination] of Object.entries(files)) {
+        await copyFile(source, `${docsDir}/${destination}`);
+    }
+};
+
+module.exports = copyMarkdownFiles;
```

Comment:
@erohmensing you do not have the last version of the script: https://github.com/RasaHQ/rasa/blob/master/docs/scripts/copy_md_files.js